gateway-js/src/index.ts

```diff
@@ -926,6 +1001,7 @@ export class ApolloGateway implements GraphQLService {
         throw Error(
           'ApolloGateway.stop does not need to be called before ApolloGateway.load',
         );
+      case 'failed to load':
```

Comment:
I think I might classify this with `initialized` where you don't stop things unless you successfully loaded? I generally assume that if I didn't successfully start something that it's not my job to stop it, and load is the equivalent of start here. The load failure happens before we start polling anyway.

Reply:
Classified with `initialized` as in it should also throw an `Error`? Right now if `failed to load`, it's basically already stopped and doesn't need to do anything, so I don't quite follow your suggestion.

Reply:
Yeah, my point is that users should *not* call `stop` unless they've successfully called `load` (directly or indirectly), and it's reasonable to throw if they have an extraneous stop call.

Reply:
That makes sense. I've combined the two cases and added a little more detail to the error message. Let me know if you think I should split the cases and use a different message for both.
4c341a7c